### PR TITLE
4 get article by id

### PR DESCRIPTION
**Pull request by Equallizerr1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -41,19 +41,50 @@ describe("API Testing", () => {
...
+				.get("/api/articles/3")
+				.expect(200)
+				.then(({ body }) => {
+					const { article } = body;
+					expect(article).toBeInstanceOf(Array);
+					expect(article.length).not.toBeLessThan(1);
+					article.forEach((article) => {
+						expect(article).toHaveProperty("author");
+						expect(article).toHaveProperty("title");
+						expect(article).toHaveProperty("article_id");
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:55]: You should be more specific here, which article id are you expecting this to be?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -41,19 +41,50 @@ describe("API Testing", () => {
...
 });
-// Implement tests later when more methods have been implemented
+
 describe("Error Testing", () => {
 	describe("Endpoint errors", () => {
-		test("Get: 404, Returns error 404 when given an invalid url", () => {
-			return request(app).get("/api/topic").expect(404).then((response) => {
-				console.log(response.body);
-			})
+		test("Err: 404, Returns error 404 when given an invalid url", () => {
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js]: This is not a 404 error - should /topic ever be an endpoint? (not to be confused with previous /topics)

File: __tests__/app.test.js [hunk 3]
```diff
@@ -41,19 +41,50 @@ describe("API Testing", () => {
...
+			return request(app).get("/api/topic").expect(404);
+		});
+		test("Err: 404, Returns error 404 when given a valid but non existant article is", () => {
```
Comment (nc-beth-mccormack) [inline __tests__/app.test.js:69]: Nice error test✅

**Review by nc-beth-mccormack — COMMENTED** [review]
Left a few comments 🙂

File: __tests__/app.test.js (hunk 2 above)
Comment (Equallizerr1) [reply __tests__/app.test.js]: was trying to get error 500 but realised now I can't test for that this way so have removed the test

**Pull request merged**